ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/config/DataEdgeConfig.java

```diff
@@ -35,4 +69,134 @@ public TaskExecutor getAsyncExecutor() {
         executor.initialize();
         return executor;
     }
+
+    @Bean
+    public Ga4ghService initGa4ghServiceImpl(final RestTemplate restTemplate,
+                                             @Value("${ega.aai.proxy.url}") final String egaAAIProxyURL,
+                                             @Value("${ega.aai.proxy.basic-authorization}") final String egaAAIProxyBasicAuth) throws URISyntaxException, MalformedURLException {
+        return new Ga4ghServiceImpl(
+                restTemplate,
+                new URL(egaAAIProxyURL),
+                egaAAIProxyBasicAuth
+        );
+    }
+
+    @Profile("!integration-test")
+    @Bean
+    public JWTService initJWTServiceImpl(@Value("${ega.aai.public.jwk.url}") final String jwkPublicKeyURL,
+                                         @Value("${ega.aai.public.jwk.connection-timeout}") final int connectTimeout,
+                                         @Value("${ega.aai.public.jwk.read-timeout}") final int readTimeout,
+                                         @Value("${ega.aai.public.jwk.size-limit-in-bytes}") final int sizeLimit,
+                                         @Value("${ega.aai.public.jwk.key-id:rsa1}") final String keyId) throws IOException, ParseException {
+        return new JWTServiceImpl(
+                load(
+                        new URL(jwkPublicKeyURL),
+                        connectTimeout,
+                        readTimeout,
+                        sizeLimit
+                ),
+                keyId);
+    }
```

Comment:
The same here, maybe it could be nice to put and enter before "throws" :) I think is a good practise to keep the exceptions together in another line when the line is too long.

Reply:
As discussed, formatting not needed.